nbs/00_utils.ipynb

```diff
@@ -12,17 +12,27 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.        for k, v in kwargs.items():
</pre><p>Delete this entire for-loop. First, <code>rng_reserve_size</code> and <code> global_seed</code> will not appear in kwargs (python will prevent it at compile time). Second, passing <code>kwargs</code> is mostly for backward compatibility. Say in the future version, we have other config attributes. We can run the code for the future version in the current version of library.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/jax-relax/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Deleted the for loop